src/Npgsql/Internal/PgTypeInfo.cs

```diff
@@ -352,11 +337,12 @@ public Type TypeToConvert
         }
     }
 
-    public required PgTypeInfo TypeInfo { get; init; }
-    public required PgConverter Converter { get; init; }
-    public required Size BufferRequirement { get; init; }
-    // Whether Converter.TypeToConvert matches the PgTypeInfo.Type, if it doesn't object apis and a downcast should be used.
-    public required bool AsObject { get; init; }
+    public PgTypeInfo TypeInfo { get; }
+    public PgConverter Converter { get; }
+    public Size BufferRequirement { get; }
+
+    /// Whether Converter.TypeToConvert matches PgTypeInfo.Type, if it doesn't object apis should be used.
```

Comment:
BTW shouldn't we be having `<summary>`?

Reply:
This is short syntax, originally from F# but C# added support a few years ago.
Basically one line like this translates to `<summary>the line</summary>`